client/src/modules/events/pages/eventPage.tsx

```diff
@@ -201,16 +202,21 @@ export const EventPage: NextPage = () => {
 
   return (
     <VStack align="flex-start">
-      <Image
-        data-cy="event-image"
-        boxSize="100%"
-        maxH="300px"
-        src={data.event.image_url}
-        alt=""
-        borderRadius="md"
-        objectFit="cover"
-        fallbackSrc="https://cdn.freecodecamp.org/chapter/brown-curtain-small.jpg"
-      />
+      {data.event.image_url !== '' ? (
+        <Box height={'300px'}>
+          <Image
+            data-cy="event-image"
+            boxSize="100%"
+            maxH="300px"
+            src={data.event.image_url === '' ? undefined : data.event.image_url}
+            alt=""
+            borderRadius="md"
+            objectFit="cover"
+            fallbackSrc="https://cdn.freecodecamp.org/chapter/brown-curtain-small.jpg"
+            fallbackStrategy="onError"
+          />
+        </Box>
+      ) : null}
```

Comment:
```suggestion
      {data.event.image_url && (
        <Box height={'300px'}>
          <Image
            data-cy="event-image"
            boxSize="100%"
            maxH="300px"
            src={data.event.image_url}
            alt=""
            borderRadius="md"
            objectFit="cover"
            fallbackSrc="https://cdn.freecodecamp.org/chapter/brown-curtain-small.jpg"
            fallbackStrategy="onError"
          />
        </Box>
      )}
```
Same principle as above.